mirrors.d/mirror.ox.ac.uk.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: mirror.ox.ac.uk
+address:
+  http: http://mirror.ox.ac.uk/sites/repo.almalinux.org/
+  https: https://mirror.ox.ac.uk/sites/repo.almalinux.org/
+  ftp: ftp://mirror.ox.ac.uk/sites/repo.almalinux.org/
+  rsync: rsync://mirror.ox.ac.uk/sites/repo.almalinux.org/
+geolocation:
+  country: UK
+  state_province: Oxfordshire
+  city: Oxford
+update_frequency: 6h
```

Comment:
On public mirrors the sync time needs to be 3h or less.  Can you run a <=3h sync time?

Reply:
OK - I think we should be ok committing to every 3h

Cheers,  Jim